### PR TITLE
SSD and Correlation Coefficient

### PR DESCRIPTION
**Pull request by hermela11**
I added ssd and correlation coefficient calculations to scil_evaluate_connectivity_pairwise_agreement_measures.py

### CODE REVIEW

File: scripts/scil_evaluate_connectivity_pairwise_agreement_measures.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        tmp_mat = load_matrix_in_any_format(filename)
+        all_matrices.append(tmp_mat / np.max(tmp_mat))
+
+    output_measures_dict = {'SSD': [], 'Correlation': []}
+    pairs = list(itertools.combinations(all_matrices, r=2))
+    for i in pairs:
+        ssd = np.sum((pairs[0] - pairs[1]) ** 2)
+        output_measures_dict['SSD'].append(ssd)
+        # only calculates for the first two arrays
+        corrcoef = np.corrcoef(all_matrices[0], all_matrices[1])
```
Comment (frheault) [inline scripts/scil_evaluate_connectivity_pairwise_agreement_measures.py:63]: This should be pairs, not all_matrices.
Also np.corrcoef expects a 1D array (everything we have is 2D) and returns a covariance matrix not a scalar.
(I think this code does not execute)

File: scripts/scil_evaluate_connectivity_pairwise_agreement_measures.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, args.in_matrices)
+    assert_outputs_exist(parser, args, [args.out_json])
```
Comment (arnaudbore) [inline scripts/scil_evaluate_connectivity_pairwise_agreement_measures.py:50]:
```suggestion
    assert_outputs_exist(parser, args, args.out_json)
```

**Review by arnaudbore — COMMENTED** [review]
Please add a test.

File: scripts/scil_evaluate_connectivity_pairwise_agreement_measures.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    assert_inputs_exist(parser, args.in_matrices)
+    assert_outputs_exist(parser, args, args.out_json)
+
+    all_matrices = []
+    for filename in args.in_matrices:
+        tmp_mat = load_matrix_in_any_format(filename)
+        all_matrices.append(tmp_mat / np.max(tmp_mat))
+
+    tmp_mat = load_matrix_in_any_format(filename)
```
Comment (frheault) [inline scripts/scil_evaluate_connectivity_pairwise_agreement_measures.py:57]: There is a `if args.single_compare:` missing here
(and replace filename by args.single_compare)

So IF single compare is used, it is loaded last.

**Review by frheault — APPROVED** [review]
After that comments its ready to merge.
@arnaudbore I will do a PR right after to add the tests since its quite 'advanced'. Is that ok?
(Right after, meaning the same day)

**Comment by frheault** [comment]
LGTM

**Review by arnaudbore — APPROVED** [review]

**Pull request merged**